src/graph/test/FetchEdgesTest.cpp

```diff
@@ -0,0 +1,207 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/test/TestEnv.h"
+#include "graph/test/TestBase.h"
+#include "graph/test/TraverseTestBase.h"
+#include "meta/test/TestUtils.h"
+
+
+namespace nebula {
+namespace graph {
+class FetchEdgesTest : public TraverseTestBase {
+protected:
+    void SetUp() override {
+        TraverseTestBase::SetUp();
+    }
+
+    void TearDown() override {
+        TraverseTestBase::TearDown();
+    }
+};
+
+TEST_F(FetchEdgesTest, base) {
+    {
+        cpp2::ExecutionResponse resp;
+        auto &player = players_["Boris Diaw"];
+        auto &serve = player.serves()[0];
+        auto &team = teams_[std::get<0>(serve)];
+        auto *fmt = "FETCH PROP ON serve %ld->%ld"
```

Comment:
why call `FETCH PROP ON`, i think  `FETCH` is enough

BTW, what about the sql-like sentence: `(better)YIELD xxx ON serve`
or (worse) `ON serve YIELD`.

Reply:
I'd like to support fetch tag/edge names in the future, so i think fetch prop is OK.